src/EnergyPlus/HeatBalanceSurfaceManager.cc

```
@@ -2791,6 +2837,12 @@ void InitSolarHeatGains(EnergyPlusData &state)
         // Calculate Exterior Incident Short Wave (i.e. Solar) Radiation on shading surfaces
         if (state.dataSurface->BuildingShadingCount || state.dataSurface->FixedShadingCount || state.dataSurface->AttachedShadingCount) {
             for (int SurfNum = state.dataSurface->ShadingSurfaceFirst; SurfNum <= state.dataSurface->ShadingSurfaceLast; SurfNum++) {
+                Real64 GndSurfReflectance = 0.0;
+                if (state.dataSurface->UseSurfPropertyGndSurfRefl(SurfNum)) {
+                    GndSurfReflectance = state.dataSurface->GroundSurfsProperty(state.dataSurface->GroundSurfsPropertyNum(SurfNum)).SurfsReflAvg;
+                } else {
+                    GndSurfReflectance = state.dataEnvrn->GndReflectance;
+                }
```

Ground reflectance value from the new Ground surfaces object `SurfaceProperty:GroundSurfaces` is used if specified, instead of the default global.